### ISSUE TITLE
[Bug]: Iron dust lost recipes

### ISSUE THREAD

**Issue by Destess** [FTBTeam/FTB-Modpack-Issues#347]
### Modpack

FTB OceanBlock

### Modpack version

1.9.0

### Describe the bug

Only Industrial Foregoing has a recipe for iron dust, when previously there were recipes from the crusher, squeezer, enrichment chamber and maybe more, but after updating there is only one recipe for iron dust

### Steps to reproduce

Search for iron dust in JEI
Press R on iron dust
Observe the single recipe

### Expected behaviour

Iron dust should have multiple recipes

### Additional information

I believe some other ores lost some crafting recipes as well, as I'm fairly certain copper cluster had more uses before

**Comment by Raidobw2** [comment]
I do not see that on my game, can you try backupping your worlds and reinstalling? You should clear the `.localCache` folder in your instances folder (if using FTB App) before reinstalling.
Please let me know if that corrects it, it seems to be bugged for you currently.
![image](https://user-images.githubusercontent.com/17010632/142719726-45e6d889-ce33-4779-80e0-a361de8d70f0.png)

**Comment by Destess** [comment]
Sorry, I never tested it in single player, and it seems as though it works fine there; on my friend's server, however, I still can't see the recipes, even after reinstalling, and his updating the server.

![image](https://user-images.githubusercontent.com/8343583/142736889-60322892-e1ad-46e0-a204-474f7e92206f.png)

**Comment by Raidobw2** [comment]
Did you reinstall your client as I suggested?

**Comment by Destess** [comment]
Yes- it only happened in multiplayer, even after my friend did the same with his server

**Comment by Raidobw2** [comment]
If your client works, then take the `config`, `mods` and `kubejs` folder from it and replace those folders inside the server. This should make recipes and everything else equal.

Let us know if that fixes it.

**Comment by Destess** [comment]
Actually, come to think of it, I think there usually is an error message when I join the server- I've just come to ignore it after seeing it enough times. I think he's said he doesn't actually have access to the server files, but that does make sense.

After joining the server, I got a text that says "KubeJS errors found [348847]! Run '/kubejs errors' for more info" and I ran both /kubejs errors and /kubejs warnings, and found a bunch of "Error parsing recipe" / "Error creating recipe" entries. 
![image](https://user-images.githubusercontent.com/8343583/142955165-93d3142b-9f1a-4ac7-9191-b4e5f76e1c25.png)
My bad, if I had just simply looked at the error I could've avoided this lol. I must've just tuned it out as white noise.
